src/krfctl/freebsd/freebsd.c

```diff
@@ -26,12 +26,13 @@
 #define LOG_FAULTS_NAME KRF_PROC_DIR "." KRF_LOG_FAULTS_FILENAME
 #define TARGETING_NAME KRF_PROC_DIR "." KRF_TARGETING_FILENAME
 
-void fault_syscall(const char *sys_name) {
+int fault_syscall(const char *sys_name) {
   const char *sys_num;
   unsigned int syscall;
 
   if (!(sys_num = lookup_syscall_number(sys_name))) {
-    errx(1, "couldn't find syscall %s", sys_name);
+    fprintf(stderr, "WARNING: couldn't find syscall %s", sys_name);
```

Comment:
Use `warnx` here, for consistency with `err`/`errx` use 😉

Reply:
Updated. Thanks